nadlogar/problems/models.py

```diff
@@ -239,3 +239,149 @@ def generate(self):
         razstavljen = sympy.factor(izraz)
 
         return {"izraz": sympy.latex(izraz), "razstavljen": sympy.latex(razstavljen)}
+
+
+class ElementiMnozice(Problem):
+    """Problem za izpis elementov množice iz podanega predpisa."""
+
+    POGOJ = ["|", "<", "<="]
+    POGOJ_LATEX = {"|": r"\mid", "<": r"\lt", "<=": r"\le"}
+
+    linearna_kombinacija = models.BooleanField(
+        "linearna kombinacija",
+        help_text="Ali naj naloga vsebuje linearno kombinacijo?",
+        choices=[(True, "Da"), (False, "Ne")],
```

Comment:
Po mojem bi morali lokalizacijo `Da/Ne` urejati ločeno, zato bi neobvezni argument `choices` pobrisal.

Reply:
To je workaround, ker če ima form samo BooleanField polja, funkcija `form.is_valid()` vrne `True` že prvič, ko pokličemo ta view (konkretni problem je v `views.py`, `create_parameters`), tako da uporabnik sploh ne vidi forme. Zato je potreben dropdown ali pa nekaj podobnega.

Reply:
Ja, `BooleanField`-i so malo broken v Djangotu...

Reply:
Ok. Razmišljam, da bi za parametre generatorjev namesto razredov iz `models.Field` uporabil bolj specifične razrede, ki bi tudi prikazali ustrezen widget, omogočali npr. izbiro razpona vrednosti, … Tako da zaenkrat pustimo, kot je, pa bomo tam reševali.